models/marts/enriched_history_operations.yml

```diff
@@ -19,28 +31,37 @@ models:
       - name: op_source_account
         description: '{{ doc("source_account") }}'
         tests:
-          - not_null
+          - not_null:
+              config:
+                where: batch_run_date > current_datetime - interval 2 day
+
 
       - name: op_source_account_muxed
         description: '{{ doc("source_account_muxed") }}'
 
       - name: transaction_id
         description: '{{ doc("transaction_id") }}'
         tests:
-          - not_null
+          - not_null:
+              config:
+                where: batch_run_date > current_datetime - interval 2 day
           - relationships:
               to: ref('stg_history_transactions')
               field: transaction_id
 
       - name: type
         description: '{{ doc("type") }}'
         tests:
-          - not_null
+          - not_null:
```

Comment:
Could we add a test that confirms that the `type <= 26`?

Reply:
Implemented through the dbt utils test "expression_is_true".